ark/analysis/visualize_test.py

```diff
@@ -65,15 +65,17 @@ def test_visualize_z_scores():
 
     with pytest.raises(FileNotFoundError):
         # trying to save on a non-existant directory
-        visualize.visualize_z_scores(z, pheno_titles, save_dir="bad_dir")
+        visualize.visualize_z_scores(z, pheno_titles, pheno_titles, save_dir="bad_dir")
 
     with tempfile.TemporaryDirectory() as temp_dir:
         # test that without save_dir, we do not save
-        visualize.visualize_z_scores(z, pheno_titles)
+        visualize.visualize_z_scores(z, pheno_titles, pheno_titles,
+                                     center_val=np.random.ranf())
         assert not os.path.exists(os.path.join(temp_dir, "z_score_viz.png"))
 
         # test that with save_dir, we do save
-        visualize.visualize_z_scores(z, pheno_titles, save_dir=temp_dir)
+        visualize.visualize_z_scores(z, pheno_titles, pheno_titles,
+                                     center_val=np.random.ranf(), save_dir=temp_dir)
```

Comment:
You should also include tests for the following:

- `center_val=None`
- `dpi` is explicitly set (currently, we're only testing `dpi=None` cases)

In both cases, you only need to include tests with a save_dir.

Reply:
Now that I think about it, this isn't necessary as these are just basic visualization control parameters that only `matplotlib` needs to worry about.